certified-connectors/Nitro/apiDefinition.swagger.json

```diff
@@ -0,0 +1,337 @@
+                          ],
+                          "properties": {
+                            "type": {
+                              "type": "string"
+                            },
+                            "description": {
+                              "type": "string"
+                            }
+                          }
+                        },
+                        "message": {
+                          "type": "object",
+                          "required": [
+                            "type",
+                            "description"
+                          ],
+                          "properties": {
+                            "type": {
+                              "type": "string"
+                            },
+                            "description": {
+                              "type": "string"
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/error"
+            }
+          }
+        }
+      }
+    },
+    "/templates/{id}/signature-requests": {
+      "post": {
+        "tags": [
+          "Template"
+        ],
+        "description": "Send a signature request using an existing template.",
+        "summary": "Send a signature request using a template",
+        "operationId": "TemplateSignatureRequest",
+        "parameters": [
+          {
+            "name": "id",
+            "type": "string",
+            "in": "path",
+            "required": true,
+            "x-ms-summary": "Template",
+            "description": "Use this option to select the template to be used for the signature request.",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "TemplateSummaries",
+              "value-path": "id",
+              "value-title": "name",
+              "value-collection": "items"
+            }
+          },
+          {
+            "name": "dynamicSchema",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "TemplateSignatureRequestSchema",
+                "parameters": {
+                  "id": {
+                    "parameter": "id"
+                  }
+                },
+                "value-path": "items"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created"
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/error"
+            }
+          }
+        }
+      }
+    },
+    "/webhooks": {
+      "x-ms-notification-content": {
```

Comment:
@diegobernardes Please remove the extra properties added here as these were not in the version of the swagger uploaded in the certification portal.

`"x-ms-notification-content": {
                "schema": {
                    "properties": {
                        "bytes": {
                            "description": "Signed document bytes",
                            "format": "byte",
                            "type": "string",
                            "x-ms-summary": "Bytes"
                        },
                        "name": {
                            "description": "Signed document name",
                            "type": "string",
                            "x-ms-summary": "Name"
                        },
                        "size": {
                            "description": "Signed document size",
                            "format": "int32",
                            "type": "integer",
                            "x-ms-summary": "Size"
                        },
                        "type": {
                            "description": "Signed document type",
                            "type": "string",
                            "x-ms-summary": "Type"
                        }
                    },
                    "required": [
                        "name",
                        "size",
                        "type",
                        "bytes"
                    ],
                    "type": "object"
                }`

to

`"x-ms-notification-content": {
                "schema": {
                    "properties": {
                        "document": {
                            "format": "byte",
                            "title": "Signed document",
                            "type": "string"
                        }
                    },
                    "required": [
                        "document"
                    ],
                    "type": "object"
                }
            }`

Reply:
Done. This change was a feature developed while the connector was being validated but it's ok after the validation ends I can push the change.